src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
I created a seperate commit for it, so it's better visible in this PR (for cherry-pick). IMO implementers are still free to add this on their own if the want to keep compatibility.